test/protocol/clients/BosonVoucher.js

```diff
@@ -504,6 +504,30 @@ describe("IBosonVoucher", function () {
       });
     });
 
+    context("getRoyaltyPercentage()", function () {
+      it("should return the royalty fee percentage", async function () {
+        // give ownership to operator
+        await bosonVoucher.connect(protocol).transferOwnership(operator.address);
+
+        royaltyPercentage = "1000"; //15%
```

Comment:
A small thing, but the number represents 10%, not 15%

Reply:
fixed